usefulUtilities/airAndWhiteboardDraw/README.md

```diff
@@ -12,28 +12,32 @@ This utility combines the best features of the whiteboard and the draw app. Any
 
 ## Releases
 ### Version 1
-SHA [80bcc13](https://github.com/highfidelity/hifi-content/commits/80bcc13)
-April 18, 2019
-- Initial Release
 
-SHA [90ebb0d](https://github.com/highfidelity/hifi-content/commits/90ebb0d)
-May 21, 2019
-- Fixed bug with paint sphere dimensions not found [JIRA 152](https://highfidelity.atlassian.net/browse/BUGZ-152)
-- Made material entity attached to paint sphere collisionless 
+SHA [](https://github.com/highfidelity/hifi-content/commits/)
+August 06, 2019
```

Comment:
is this the version number?
what's the version number associated with this release?

Reply:
There were no version numbers on the other releases (aside from the first), only dates, so I did not add one. Should I add versions to them all?

Reply:
hmm - how have you/we been tracking which code is associated with previous whiteboard releases?

yes, please add accurate version numbers to all past releases. the version numbers for past releases should match the version numbers already released on S3.